docs/UserGuide.md

```diff
@@ -24,9 +24,9 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   1. view client command
-   2. add client command
-   3. delete client command
+   1. `viewClient`: View a particular client
+   2. `addClient`: Add a client
+   3. `deleteClient`: Delete client
```

Comment:
Can consider adding a link to the method

ie

[`viewClient` ](#viewClient)

Just a suggestion, can do it sometime later. 
I will just approve this now cause I think no big issues :)

Reply:
ok, will keep in mind to add this in next iteration.